src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -71,8 +71,9 @@ static char* create_standard_memory(size_t size) {
 
   // commit memory
   if (!os::commit_memory(mapAddress, size, !ExecMem)) {
-    if (PrintMiscellaneous && Verbose) {
-      warning("Could not commit PerfData memory\n");
+    if (log_is_enabled(Debug, perf)) {
+      LogStreamHandle(Debug, perf) log;
+      log.print_cr("Could not commit PerfData memory\n");
```

Comment:
For simple usages like this you can simplify.
```suggestion
      log_debug(perf)("Could not commit PerfData memory");
```
also note to get rid of the explicit `\n` in the strings.